03/app.js

```diff
@@ -0,0 +1,15 @@
+//nie działa zatrzymanie EventList..
+const button = document.querySelectorAll('button');
+
+button.forEach(function(el) {
+    el.addEventListener('click', transformButton); 
+    // zatrzymanie po kliknięciu w przycisk
+    if(el.innerText === 'clicked') {
+        el.removeEventListener('click', transformButton);
+    }
```

Comment:
Usuwanie zdarzenia powinno mieć miejsce wew. funkcji `transofrmButton` ponieważ to jej zawartość jest uruchamiana podczas zdarzenia.

PS. Pytanie czy musimy w ogóle robić warunek - tutaj mamy, że zawsze po pierwszym kliknięciu mamy usunąć nasłuchiwanie - nie jest istotne jaka jest treść.

Reply:
ok, już działa. Usunęłam warunek.